src/js/main.js

```diff
@@ -2,6 +2,7 @@
 
 import Icons from '../elements/icons/Icons';
 import Feed from '../objects/feed/Feed';
+import Chart from '../objects/chart/chart';
```

Comment:
We need to import the chart object, this is basically a wrapper class for the Vue app that will display the vue object.